lib/utils/ember.js

```diff
@@ -129,6 +129,9 @@ function isObserverProp(node) {
 }
 
 function isComputedProp(node) {
+  if (utils.isMemberExpression(node.callee) && utils.isCallExpression(node.callee.object)) {
+    return isModule(node.callee.object, 'computed') && utils.isIdentifier(node.callee.property);
```

Comment:
We were speaking about white listing four methods that are supported on CPs (`meta`, `property`, `readOnly` and `volatile` regarding https://www.emberjs.com/api/ember/2.14.0/classes/Ember.ComputedProperty). Are you going to add them?